modules/ROOT/pages/availability_scaling/index.adoc

```diff
@@ -0,0 +1,140 @@
+
+==== Separating User Data
+
+The first consideration in terms of availability, flexibility and scalability can be made with respect where to store user data. It is **highly recommended** to physically separate user data from the instance and store them on a shared filesystem. See the xref:prerequisites/index.html#filesystems-and-external-storage[Filesystems and Shared Storage] section for supported types.
+
+image::availability_scaling/ocis_with_data_server.svg[width=300]
+
+Summary::
+Also this scenario is absolutely valid for smaller environments or testing purposes. In case of a failure, is is much easier to fix the defect because of separated components. *Scalability is on a very low level*.
+
+=== Forms of High Availability
+
+High Availability with its flavours active-active / active-passive layout or clustering provide redundancy by eliminating the node as a single point of failure. They are able to provide availability in these scenarios:
+
+* Software crashes, either due to operating system failure or unrecoverable applications.
+* Hardware failures, including storage hardware, CPU, RAM, network interfaces, etc.
+* Virtualization host system failures, including unplanned and scheduled maintenance.
+* Logically or physically severed network, if the failover appliance is on a separate network not impacted by the failure.
+* Regular planned node maintenance.
+
+// do not delete, dummy necessary to have the summary not part of the above list item - antora 2.3 bug
+
+Summary::
+As there are many different ways to solve a required scenario, ownCloud support _cant_ give advice for a particular solution that fits your needs but may be able to help to get the required oCIS component ready to run.
++
+For availability, storing user files is by nature mandatory to be on xref:prerequisites/index.html#filesystems-and-external-storage[shared storage] to be accessible by the nodes and/or services.
+
+==== Classic High Availability
+
+When using the classic form of high availability (HA), you can either make a setup where the nodes are both active (active-active) or one node is spare, waiting for a failure to occur (active-passive). With an A-A setup, a loadbalancer is needed in front of the nodes and in case of a failure, the remaining node has to take all the load additionally the failed one served. 
+
+image::availability_scaling/a_a_ha_with_data_server.svg[width=310]
+
+Summary::
+This usecase can be considered if hardware availability is the primary obective. When using an active-active configuration, each node is addressed by a loadbalancer for load distribution. This requires that the nodes have the same setup and services are bound to the nodes. *Scalability is hardware dependent and in case of an active-passive setup, you can even get reduced scalability*.
```

Comment:
```suggestion
This use case can be considered if hardware availability is the primary objective. When using an active-active configuration, each node is addressed by a load balancer for load distribution. This requires that the nodes have the same setup and services are bound to the nodes. *Scalability is hardware dependent and in case of an active-passive setup, you can even get reduced scalability*.
```